test_run.lua

```diff
@@ -163,15 +163,15 @@ local function wait_fullmesh(self, servers)
                 log.info("%s: bootstrapped", server)
                 break
             end
-            local info = self:eval(server, "box.info")
+            self:eval(server, "box.info")
             fiber.sleep(0.01)
         end
         -- wait all for full mesh
         for _, server2 in ipairs(servers) do
             if server ~= server2 then
                 log.info("%s -> %s: waiting for connection", server2, server)
                 while true do
-                    local cmd = wait_fullmesh_cmd:format(server_id)
+                    wait_fullmesh_cmd:format(server_id)
```

Comment:
`string.format()` returns a new string: it does not change a string in place. Strings in Lua are immutable.

Reply:
Reverted change.